src/sonic_ax_impl/mibs/ietf/rfc1213.py

```diff
@@ -126,6 +133,12 @@ def get_counter(self, sub_id, table_name):
         :param table_name: the redis table (either IntEnum or string literal) to query.
         :return: the counter for the respective sub_id/table.
         """
+        if sub_id in self.oid_lag_name_map:
+            counter_value = 0
+            for lag_member in self.lag_name_if_name_map[self.oid_lag_name_map[sub_id]]:
+                counter_value += self.get_counter(mibs.get_index(lag_member), table_name)
```

Comment:
It will be truncated to 32 bit in upper layer. Changed code to truncate it explicitly.